programs/eosc/main.cpp

```diff
@@ -21,6 +22,8 @@
 
 #include <fc/io/fstream.hpp>
 
+#include "CLI11.hpp"
```

Comment:
Comment with verison of CLI11 used might be helpful, I assume version 1.1.

Reply:
At this point we're using the HEAD version, as 1.1 contained a bug [which I reported](https://github.com/CLIUtils/CLI11/issues/23) and he fixed, slated for release in version 1.2. I've included the header in our source distribution so there's no worries about having to go find the right version.